pranshugupta54/digitomize
# fix: filter div fixed and updated

## What does this PR do?

This PR updates the filter for the contests page as per the requirements given.

Fixes #73 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)


## How should this be tested?

go to the contest page select dropdown add some things from the list check also try deleting them with a cross.

## Mandatory Tasks
